networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -24,6 +24,11 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 
 * Applying a YAML manifest: You can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
 
+[NOTE]
+====
+When you install {product-title} on {rh-openstack-first} by using Open Virtual Network (OVN), a worker node has an additional network with a subnet that has no DNS server address. You must set a DNS server address as `0.0.0.0` in the `/etc/resolv.conf` file on the worker node.
```

Comment:
This is not correct. When deploying nodes with multiple interfaces on certain configuration of RHOSP (RHOSP with the OVN SDN), it is possible that the DNS setting coming from the secondary interface takes precedence over the DNS setting coming from the primary one. This becomes problematic if the DNS server can't resolve internal records.
In that case, _the OpenStack admin_ should change the DNS setting for the subnet, or remove DNS settings for the subnet:
```
openstack subnet set --no-dns-nameservers <subnet_id>
```

https://docs.openstack.org/neutron/latest/admin/config-dns-res.html#case-1-each-virtual-network-uses-unique-dns-resolver-s

See also https://issues.redhat.com/browse/OCPBUGS-3998 for a manifestation of this issue.

Reply:
@itzikb-redhat please confirm my understanding of the issue is correct.

Reply:
@mandre I updated the PR- could you please check?

Reply:
@itzikb-redhat Could you PTAL too?